notebooks/MAST/PanSTARRS/PS1_DR2_TAP/PS1_DR2_TAP.ipynb

```diff
@@ -0,0 +1,899 @@
+    "detection_TAP_results['mag'] = -2.5*np.log10(detection_TAP_results['psfFlux']) + 8.90\n",
+    "detection_TAP_results['dmag'] = detection_TAP_results['mag']-detection_TAP_results['magmean']\n",
+    "\n",
+    "detection_TAP_results"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "t = detection_TAP_results['obsTime']\n",
+    "dmag = detection_TAP_results['dmag']\n",
+    "xlim = np.array([t.min(),t.max()])\n",
+    "xlim = xlim + np.array([-1,1])*0.02*(xlim[1]-xlim[0])\n",
+    "ylim = np.array([dmag.max(),dmag.min()])\n",
+    "ylim = ylim + np.array([-1,1])*0.02*(ylim[1]-ylim[0])\n",
+    "\n",
+    "pylab.rcParams.update({'font.size': 14})\n",
+    "pylab.figure(1,(10,10))\n",
+    "for i, filter in enumerate([b'g',b'r',b'i',b'z',b'y']):\n",
+    "    pylab.subplot(511+i)\n",
+    "    w = np.where(detection_TAP_results['filterType']==filter)[0]\n",
+    "    pylab.plot(t[w],dmag[w],'-o')\n",
+    "    magmean = detection_TAP_results['magmean'][w[0]]\n",
+    "    pylab.ylabel('{} [mag - {:.2f}]'.format(filter,magmean))\n",
+    "    pylab.xlim(xlim)\n",
+    "    pylab.ylim(ylim)\n",
+    "    if i==0:\n",
+    "        pylab.title(objname)\n",
+    "pylab.xlabel('Time [MJD]')\n",
+    "pylab.tight_layout()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Plot versus phase using known period\n",
+    "\n",
+    "Eclipsing binaries basically vary by same amount in all filters since it is a geometrical effect, so combine the data into a single light curve.  Wrap using known period and plot versus phase."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "period = 2.2834698\n",
+    "bjd0 = 54999.599837\n",
+    "t = ((detection_TAP_results['obsTime']-bjd0) % period) / period\n",
+    "dmag = detection_TAP_results['dmag']\n",
+    "w = np.argsort(t)\n",
+    "t = t[w]\n",
+    "dmag = dmag[w]\n",
+    "xlim = np.array([t.min(),t.max()])\n",
+    "xlim = xlim + np.array([-1,1])*0.02*(xlim[1]-xlim[0])\n",
+    "ylim = np.array([dmag.max(),dmag.min()])\n",
+    "ylim = ylim + np.array([-1,1])*0.02*(ylim[1]-ylim[0])\n",
+    "\n",
+    "pylab.rcParams.update({'font.size': 14})\n",
+    "pylab.figure(1,(10,6))\n",
+    "pylab.plot(t,dmag,'-o')\n",
+    "pylab.xlim(xlim)\n",
+    "pylab.ylim(ylim)\n",
+    "pylab.xlabel('Phase')\n",
+    "pylab.ylabel('Delta magnitude from mean [mag]')\n",
+    "pylab.title(objname)\n",
+    "pylab.tight_layout()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Repeat search for another eclipsing binary KIC 8153568"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "objname = 'KIC 8153568'\n",
+    "ra, dec = resolve(objname)\n",
```

Comment:
If you use the mast astroquery resolver this would change to:
```
coords = Mast._resolve_object(objname)
ra,dec = coords.ra.value, coords.dec.value
```